src/components/Header/HeaderNav/HeaderNav.js

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
 
 const HeaderNav = (props) => {
+  const navItems = props.nav;
   return (
     <nav>
       <ul>
-        <li>{props.user.Email}</li>
+        {navItems.map((item, index) => (
+          <li key={index}>
+            <a className={item.isSelected ? 'active' : ''} href={item.url}>
+              {item.title}
+            </a>
+            {item.subNav ? (
+              <ul>
+                {item.subNav.map((itemSubNav, index) => (
+                  <li key={index}>
+                    <a className={itemSubNav.isSelected ? 'active' : ''} href={itemSubNav.url}>
```

Comment:
Las comillas hay que ponerlas o JSX las pone solo?

```suggestion
                    <a className={itemSubNav.isSelected ? 'active' : ''} href='{itemSubNav.url}'>
```

Reply:
si pones las comillas te lo toma como un string

Reply:
Joya, gracias por la aclaración, tenemos que aprender mucho de esta notación.